bedrock/privacy/templates/privacy/notices/firefox-2025.html

```diff
@@ -14,6 +14,11 @@
   {% endif %}
 {% endblock %}
 
+{% block extrahead %}
```

Comment:
👍 definitely a better place, it's redirectly without the flash of content I was seeing from the bottom-of-page JS block